app/models/host_initiator.rb

```diff
@@ -0,0 +1,19 @@
+class HostInitiator < ApplicationRecord
+  include NewWithTypeStiMixin
```

Comment:
You have this here but no `:type` column in https://github.com/ManageIQ/manageiq-schema/pull/537/files#diff-c0fd360c779d43e4b9d848b76609693d0c69f581d56efb604ec220d4e0e6535cR3

Do you want this to be subclassed by your provider or not?

Reply:
Since this is a glorified join table I'm going to assume you don't want STI and this should be dropped

Reply:
done